src/autopas/containers/cellPairTraversals/SlicedTraversal.h

```diff
@@ -83,14 +83,23 @@ inline void SlicedTraversal<ParticleCell, PairwiseFunctor, useSoA, useNewton3>::
 
   // split domain across its longest dimension
 
-  auto numSlices = autopas_get_max_threads();
-  // find dimension with most cells
+  auto numSlices = (size_t)autopas_get_max_threads();
+  auto minSliceThickness = this->_cellsPerDimension[_dimsPerLength[0]] / numSlices;
+  if (minSliceThickness < 2) {
+    AutoPasLog(debug, "Sliced travesal only using {} threads because number of cells is too small.", numSlices)
```

Comment:
```suggestion
    AutoPasLog(debug, "Sliced traversal only using {} threads because the number of cells is too small.", numSlices)
```

Reply:
also: the correct calculation of the used slices is AFTER this debug message -> this line should go after line 91.

Reply:
fixed 078ef728302b9d94ec5c6b40626826f56e75279c